//package com.sincetimes.redwar.game.comnunicate.protocol;

message RceDeployUnits
{
	optional int32 planetId = 1;//": "1", 
    optional int32 x = 2;//": 732,
    repeated int32 unitsIds = 3;
    optional int32 hangarSid = 4;//": 40,
    optional int32 y = 5;//": 1165,
    repeated int32 skuSD = 6;
    repeated int32 skuE = 7;
    optional int32 millis = 8;//": 73524,
    repeated int32 skuSWT = 9;
    repeated string unitsSkus = 10;
}

